src/Sylius/Bundle/InstallerBundle/Behat/CliContext.php

```diff
@@ -0,0 +1,166 @@
+<?php
+
+/*
+ * This file is part of the Sylius package.
+ *
+ * (c) Paweł Jędrzejewski
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Sylius\Bundle\InstallerBundle\Behat;
+
+use Sylius\Bundle\InstallerBundle\Command\SetupCommand;
+use Sylius\Bundle\ResourceBundle\Behat\DefaultContext;
+use Symfony\Bundle\FrameworkBundle\Console\Application;
+use Symfony\Component\Console\Helper\DialogHelper;
+use Symfony\Component\Console\Tester\CommandTester;
+use Symfony\Component\EventDispatcher\EventDispatcher;
+
+/**
+ * @author Magdalena Banasiak <magdalena.banasiak@lakion.com>
+ */
+class CliContext extends DefaultContext
+{
+    /**
+     * @var Application
+     */
+    private $application;
+
+    /**
+     * @var CommandTester
+     */
+    private $tester;
+
+    /**
+     * @var DialogHelper
+     */
+    private $dialog;
+
+    /**
+     * @var SetupCommand
+     */
+    private $command;
+
+    /**
+     * @var array
+     */
+    private $inputChoices = array(
+        'currency' => '',
+        'name' => ' Name',
+        'surname' => ' Surname',
+        'e-mail' => ' test@email.com',
+        'password' => ' pswd',
+        'confirmation' => ' pswd',
+    );
+
+    /**
+     * @When /^I run a command "([^"]+)"$/
+     */
+    public function iRunACommand($name)
+    {
+        $this->application = new Application($this->getKernel());
+        $this->application->add(new SetupCommand());
+
+        $this->command = $this->application->find($name);
+        $this->tester = new CommandTester($this->command);
+
+        $this->iExecuteCommandWithInputChoices($name);
+    }
+
+    /**
+     * @Then /^I should see output "(.+)"$/
+     */
+    public function iShouldSeeOutput($text)
+    {
+        \PHPUnit_Framework_Assert::assertContains($text, $this->tester->getDisplay());
+    }
+
+    /**
+     * @Given I do not provide a currency
+     */
+    public function iDoNotProvideCurrency()
+    {
+        $this->inputChoices['currency'] = '';
+    }
+
+    /**
+     * @Given I do not provide a name
+     */
+    public function iDoNotProvideName()
+    {
+        array_splice($this->inputChoices, 1, 0, '');
```

Comment:
@michalmarcinkowski No, because we do not want to leave the name blank, but to first give blank input and then continue running the command, that is why we only insert blank input into the array. :)